Bash_scripts/HW20/group_vars/linux/linux_var.yml

```diff
@@ -0,0 +1,3 @@
+#Linux vars
+ansible_user: vagrant
```

Comment:
`ansible_user` and `ansible_ssh_private_key_file` can be set up for all hosts, not for each group

Reply:
done. left common variables for all hosts in the Linux group